bciers/apps/registration1/tests/shared/DataGrid.test.tsx

```diff
@@ -103,10 +99,14 @@ describe("The DataGrid component", () => {
     act(() => {
       screen.getAllByLabelText("Sort")[0].click();
     });
-    expect(mockReplace).toHaveBeenCalledTimes(2); // two times because we set params twice, once for sort_field, once for sort_order?
```

Comment:
I'm not entirely sure what is happening here but I believe the first call of nextjs app router replace was triggering a hard refresh since it doesn't support shallow routing so it was being called and re-rendering the datagrid twice.